source/client/remote_process_impl.cc

```diff
@@ -54,5 +54,11 @@ bool RemoteProcessImpl::run(OutputCollector& collector) {
   return false;
 }
 
+bool RemoteProcessImpl::requestExecutionCancellation() {
+  ENVOY_LOG(error, "Remote process cancellation not supported yet");
+  // TODO(XXX): Send a cancel request to the gRPC service.
```

Comment:
Did we want to replace XXX with a value?

Reply:
I filed https://github.com/envoyproxy/nighthawk/issues/380 (and updated `XXX` to `#380`) in the changes I'm about to push.